package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "connector-registry-storage-adapter",
+  "version": "1.0.0",
+  "description": "Adapter for Connector Registry github storage",
+  "main": "dist/index.js",
+  "scripts": {
+    "build": "tsc --build",
+    "clean": "tsc --build --clean",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/devx-services/connector-registry-storage-adapter.git"
+  },
+  "author": "VladimirZaets",
+  "license": "ISC",
```

Comment:
Shouldn't it be `"license": "Apache-2.0"`?

Reply:
Done